crates/nu-command/src/formats/to/nuon.rs

```diff
@@ -42,14 +42,20 @@ impl Command for ToNuon {
         call: &Call,
         input: PipelineData,
     ) -> Result<PipelineData, ShellError> {
-        let raw = call.has_flag(engine_state, stack, "raw")?;
-        let tabs: Option<usize> = call.get_flag(engine_state, stack, "tabs")?;
-        let indent: Option<usize> = call.get_flag(engine_state, stack, "indent")?;
+        let style = if call.has_flag(engine_state, stack, "raw")? {
+            nuon::ToStyle::Raw
+        } else if let Some(t) = call.get_flag(engine_state, stack, "tabs")? {
+            nuon::ToStyle::Tabs(t)
+        } else if let Some(i) = call.get_flag(engine_state, stack, "indent")? {
+            nuon::ToStyle::Spaces(i)
```

Comment:
Future improvment: Passing them in combination should result in an error.

Reply:
should be taken care of by
- https://github.com/nushell/nushell/pull/12595